packages/editor/src/client/atoms/comment/commentUser.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {useState} from 'react'
+import {useTranslation} from 'react-i18next'
+import {Col, Drawer, Form, Row} from 'rsuite'
+
+import {FullCommentFragment, FullUserFragment, ImageRefFragment} from '../../api'
+import {ImageSelectPanel} from '../../panel/imageSelectPanel'
+import {ChooseEditImage} from '../chooseEditImage'
+import {UserSearch} from '../searchAndFilter/userSearch'
+
+interface CommentUserProps {
+  comment?: FullCommentFragment
+  setComment(comment: FullCommentFragment): void
+}
+
+export function CommentUser({comment, setComment}: CommentUserProps) {
+  const {t} = useTranslation()
+  const [open, setOpen] = useState<boolean>(false)
```

Comment:
Just a small nitpick, type inference should do the trick here

```suggestion
  const [open, setOpen] = useState(false)
```